main.py

```diff
@@ -17,7 +17,8 @@ def cli():
         question = input("Enter you question; just press enter for ending the session: ")
         if question.lower() == "":
             break
-        print(docqa.answer_query(question)["choices"][0]["message"]["content"])
```

Comment:
this helps in better readability but is not necessarily needed. Since if statement always breaks out of the loop. Will merge anyways.